### PR TITLE
[k8s] Rename `client` to `p1` in k8s localnet

### PR DESCRIPTION
**Pull request by h5law**
## Description

This PR changes the binary in the k8s `localnet_shell` to use the same name as it would get from `make build`: `p1`. This is in response to the following message:

image

## Issue

Fixes N/A

## Type of change

Please mark the relevant option(s):

- [ ] New feature, functionality or library
- [ ] Bug fix
- [x] Code health or cleanup
- [ ] Major breaking change
- [ ] Documentation
- [ ] Other

## List of changes

- Change `client` binary to be called `p1`

## Testing

- [x] `make develop_test`; if any code changes were made
- [x] `make test_e2e` on [k8s LocalNet](https://github.com/pokt-network/pocket/blob/main/build/localnet/README.md); if any code changes were made
- [ ] `e2e-devnet-test` passes tests on [DevNet](https://pocketnetwork.notion.site/How-to-DevNet-ff1598f27efe44c09f34e2aa0051f0dd); if any code was changed
- [x] [Docker Compose LocalNet](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md); if any major functionality was changed or introduced
- [x] [k8s LocalNet](https://github.com/pokt-network/pocket/blob/main/build/localnet/README.md); if any infrastructure or configuration changes were made

## Required Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added, or updated, [`godoc` format comments](https://go.dev/blog/godoc) on touched members (see: [tip.golang.org/doc/comment](https://tip.golang.org/doc/comment))
- [x] I have tested my changes using the available tooling
- [ ] I have updated the corresponding CHANGELOG

### If Applicable Checklist

- [ ] I have updated the corresponding README(s); local and/or global
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added, or updated, [mermaid.js](https://mermaid-js.github.io) diagrams in the corresponding README(s)
- [ ] I have added, or updated, documentation and [mermaid.js](https://mermaid-js.github.io) diagrams in `shared/docs/*` if I updated `shared/*`README(s)

### CODE REVIEW

File: app/client/cli/cmd.go
Comment (dylanlott) [inline app/client/cli/cmd.go:13]: A lot of infra relies on this path indirectly and any change will need to be thoroughly vetted through that process. I'm not opposed to a name change on good grounds, but I'm not understanding how this is worthwhile. In the past, `pocket` has been avoided because it wasn't descriptive in context, so `client` could certainly be improved upon. `p1` is similarly vague, I would argue. Naming is a hard problem 😅

**Review by dylanlott — CHANGES_REQUESTED** [review]

File: app/client/cli/cmd.go
Comment (Olshansk) [reply app/client/cli/cmd.go:13]: ## Re validation

Good point. @h5law Can you sanity check (manual with localnet, make test_e2e, etc) just to make sure it passes all the checkboxes we have in the new PR template.

@dylanlott Do you think that would be enough?

## Re Naming

`p1` was actually selected very carefully and what we use in `build.mk`. I believe we should continue using it

Screenshot 2023-05-17 at 1 49 10 PM

Comment (h5law) [reply app/client/cli/cmd.go:13]: Updated `e2e` tests to use the new `p1` binary name in k8s 👍🏼

**Review by dylanlott — COMMENTED** [review]
Ah, with the above context on `p1` I see the intent more clearly. Will approve once DevNet flows are tested. 👍

File: app/client/cli/cmd.go
Comment (Olshansk) [reply app/client/cli/cmd.go:13]: Once the PR (#737) from @okdas merges in, you can just add the `e2e-devnet-test` label like we mention in [1]

[1] https://github.com/pokt-network/pocket/wiki/Development-Readme#github-labels

**Review by Olshansk — CHANGES_REQUESTED** [review]
@h5law I tried manually run the workflow on your PR by going to the [actions tab](https://github.com/pokt-network/pocket/actions) but it wasn't available.

Can you try merging with the latest main and repushing to see if that works?
Screenshot 2023-05-19 at 6 07 50 PM


---

@okdas Have you confirmed this is working since you're latest merge to main?

File: build/Dockerfile.debian.dev
Comment (h5law) [inline build/Dockerfile.debian.dev:56]: @okdas ⭐

**Comment by h5law** [comment]
Missed the dockerfile, now thats updated the e2e tests pass locally in k8s and on devnet: ty @okdas <3 for putting up with my questions ahah

**Review by dylanlott — APPROVED** [review]
LGTM

**Pull request merged**
